data/World/Spirit Temple MQ.json

```diff
@@ -66,7 +66,8 @@
                 (Small_Key_Spirit_Temple, 6) and can_play(Zeldas_Lullaby) and Megaton_Hammer",
             "Mirror Shield Hand": "
                 (Small_Key_Spirit_Temple, 5) and can_play(Song_of_Time) and
-                (logic_lens_spirit_mq or can_use(Lens_of_Truth))"
+                (logic_lens_spirit_mq or can_use(Lens_of_Truth))",
+            "Spirit Temple Boss Platform": "spirit_temple_shortcuts"
```

Comment:
How is adult making it onto the boss platform from entering this room? Vanilla needs longshot but I don't remember MQ not needing it.

Reply:
Adult needs longshot to even get into main room.
Hookshot can work as well, would be a solid trick inclusion I think.

Reply:
Oh, I see the longshot is already included. Thanks!